test/cipherlist_test.c

```diff
@@ -136,18 +136,18 @@ static int test_default_cipherlist(SSL_CTX *ctx)
     OPENSSL_assert(ciphers != NULL);
     num_expected_ciphers = OSSL_NELEM(default_ciphers_in_order);
     num_ciphers = sk_SSL_CIPHER_num(ciphers);
-    if (num_ciphers != num_expected_ciphers) {
-        fprintf(stderr, "Expected %d supported ciphers, got %d.\n",
-                num_expected_ciphers, num_ciphers);
+    if (!TEST_int_eq(num_ciphers, num_expected_ciphers)) {
```

Comment:
Looking at these, I had another thought. Can we define the macros as

```
#define TEST_int_eq(a, b) test_int_eq(__FILE__, __LINE__, #a, #b, a, b)
```

i.e., stringify the value(s) under test and print upon failure. Then the error line contains enough information that you could mostly scrap the `TEST_info` lines.